gourmet/defaults/defaults.py

```diff
@@ -1,6 +1,8 @@
 import locale, os
+from typing import Optional
+from .abstractLang import AbstractLanguage
 deflang = 'en'
-lang = None
+lang: AbstractLanguage
```

Comment:
I'm not familiar with this syntax, can you shed some light?

Reply:
This is just a type definition. If lang is ever `None`, we need an `Optional[Language]` which defeats the purpose. But I haven't noticed, that if all imports fail `if not lang:` gives a `NameError`. I will fix that.

We will probably, at some point, need to change code that is something like `x = x[0]` as this gives a stupid type for `x`.